### DISCUSSION TITLE
Mod the icons in "Assign Status Effects" and add new one.

### DISCUSSION THREAD

**Discussion by DarthBalrog** [DFreds/dfreds-convenient-effects#352] [Q&A] [answered]
Hi,
How to create new Status Effects in "Assign Status Effects"?
Is it possible to remove some Status Effects from the list in "Assign Status Effects"?
Is it possible to mod the icon of the Status Effects in the list "Assign Status Effects"?
If so, how?
Than you!

**Comment by DFreds** [comment, marked as answer]
Are you talking about the status effects that are listed on the token? It's not possible anymore do to the level of difficulty in doing so. FoundryVTT doesn't support this as a moddable thing, making it very brittle
